test/RelayServer.webpack.test.js

```diff
@@ -0,0 +1,27 @@
+// eslint-disable-next-line no-unused-vars
+import childProcess from 'child_process'
+import path from 'path'
+import fs from 'fs'
+
+describe.only('RelayServer-webpack', () => {
+  let oneFileRelayer
+  before('create webpack', function () {
+    this.timeout(5000)
+    const jsrelayDir = path.join(__dirname, '..', 'jsrelay')
+    fs.rmdirSync(path.join(jsrelayDir, 'dist'), { recursive: true })
+    childProcess.execSync('npx webpack', { cwd: jsrelayDir })
+    oneFileRelayer = path.join(jsrelayDir, 'dist', 'relayserver.js')
+  })
+
+  it('should launch (and say "missing address")', () => {
+    try {
+      childProcess.execSync('node ' + oneFileRelayer, { encoding: 'ascii' })
+      assert.fail('should throw')
+    } catch (e) {
+      assert.match(e.toString(), /missing --RelayHubAddress/)
+    }
+    // just verify it launches
+  })
+
+  it('should test it can actually work')
```

Comment:
it really should

Reply:
That's a separate issue to repeat  server suite test with this one-file relayer.
this test only checks that the webpack was successful (in the cleanroom of CI)

the point is that we don't bundle this 4MB json file into our package (its a merge of all installed packages used by the server) - though it can be exported as a single-file-package of a server.